src/Primitives/src/StringSegment.cs

```diff
@@ -209,7 +209,7 @@ public static bool Equals(StringSegment a, StringSegment b, StringComparison com
         /// Checks if the specified <see cref="string"/> is equal to the current <see cref="StringSegment"/>.
         /// </summary>
         /// <param name="text">The <see cref="string"/> to compare with the current <see cref="StringSegment"/>.</param>
-        /// <returns><code>true</code> if the specified <see cref="string"/> is equal to the current <see cref="StringSegment"/>; otherwise, <code>false</code>.</returns>
+        /// <returns><c>true</c> if the specified <see cref="string"/> is equal to the current <see cref="StringSegment"/>; otherwise, <c>false</c>.</returns>
```

Comment:
```suggestion
        /// <returns><see langword="true" /> if the specified <see cref="string"/> is equal to the current <see cref="StringSegment"/>; otherwise, <see langword="false" />.</returns>
```